lib/OpenQA/WebAPI/Controller/Main.pm

```diff
@@ -162,5 +163,12 @@ sub parent_group_overview {
     $self->group_overview('JobGroupParents', 'main/parent_group_overview');
 }
 
+sub changelog {
+    my ($self) = @_;
+    my $changelog = decode('utf8', `rpm -q --changelog openQA`);
```

Comment:
Of course this needs error handling (that's why it is still WIP). But I'll wait with it till the change log is packaged.